Testscripts/Linux/disableEnablePci.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+
+# Source utils.sh
+. utils.sh || {
+    echo "ERROR: unable to source utils.sh!"
+    echo "TestAborted" > state.txt
+    exit 2
+}
+UtilsInit
+
+if [ ! -z "$expectedNvme" ] && [ "$expectedNvme" -gt 0 ]; then
+    # Count the NVME disks
+    disk_count=$(ls -l /dev | grep -w nvme[0-9]$ | awk '{print $10}' | wc -l)
+    if [ "$disk_count" -ne "$expectedNvme" ]; then
+        LogErr "NVME disks count does not match expected inside the VM, expected $expectedNvme found $disk_count"
+        SetTestStateFailed
+        exit 0
+    fi
+fi
+
+if [ ! -z "$expectedSriov" ] && [ "$expectedSriov" -gt 0 ]; then
+    #count SR-IOV NICs
+    nics_count=$(find /sys/devices -name net -a -ipath '*vmbus*' | grep pci | wc -l)
```

Comment:
https://github.com/koalaman/shellcheck/wiki/SC2126

Reply:
moved to grep -c